pkgs/tools/misc/dooit/default.nix

```diff
@@ -5,34 +5,27 @@
 
 python3.pkgs.buildPythonApplication rec {
   pname = "dooit";
-  version = "1.0.1";
+  version = "2.0.1";
   format = "pyproject";
 
   src = fetchFromGitHub {
     owner = "kraanzu";
     repo = pname;
     rev = "v${version}";
-    hash = "sha256-7a6xoqbAmnGVUVppQTSo4hH44XFCqBnF7xO7sOVySY0=";
+    hash = "sha256-iQAGD6zrBBd4fJONaB7to1OJpAJUO0zeA1xhVQZBkMc=";
   };
 
-  # Required versions not available
-  pythonRelaxDeps = [
-    "textual"
-    "tzlocal"
-  ];
-
   nativeBuildInputs = with python3.pkgs; [
     poetry-core
-    pythonRelaxDepsHook
   ];
 
   propagatedBuildInputs = with python3.pkgs; [
-    textual
+    appdirs
     pyperclip
+    python-dateutil
     pyyaml
-    dateparser
+    textual
     tzlocal
-    appdirs
   ];
 
   # No tests available
```

Comment:
The  package is missing a `pythonImportsCheck` test

Reply:
oh this is not a module

Reply:
oh this is not a module nvm that.

Reply:
Please add a https://nixos.org/manual/nixpkgs/stable/#tester-testVersion

Reply:
Pushed a test, we'll see what ofborg sees. Locally it still says 2.0.0 from the doot --version

Reply:
The dev forgot to bump the version https://github.com/kraanzu/dooit/compare/v2.0.1...main and only did it after the release, use `substituteInPlace` in `postPatch` to fix it

Reply:
Ah, lame. I'll do it when I get back tonight. Thanks for catching that